controllers/jobs.js

```diff
@@ -16,11 +25,31 @@ const createJob = async (req, res) => {
 }
 
 const updateJob = async (req, res) => {
-    res.send('update job');
+    const { body:{company, position}, user: {userId}, params: {id:jobId} } = req;
+
+    if (company === '' || position === '') {
+        throw new BadRequestError('Please provide a company and position');
+    }
+
+    const job = await Job.findByIdAndUpdate({ _id:jobId, createdBy:userId }, req.body, {new:true, runValidators:true});
```

Comment:
Similarly, in large function calls, it's nice to split up arguments that you're passing for readability:
```node.js
const job = await Job.findByIdAndUpdate(
    { _id: jobId, createdBy: userId }, 
    req.body, 
    { new: true, runValidators: true }
);
```